keras/layers/preprocessing/image_preprocessing.py

```diff
@@ -1359,6 +1360,34 @@ def get_config(self):
     return dict(list(base_config.items()) + list(config.items()))
 
 
+@keras_export('keras.layers.experimental.preprocessing.Keras3DAugmentation')
+class Keras3DAugmentation(base_layer.Layer):
```

Comment:
This layer appears near identical in functionality to `layers.TimeDistributed`. Maybe you should just use that instead.

Reply:
I just noticed and I agree. I think we should close the pull request and issue. Thanks!